analysis/warriorarms/src/modules/features/RageTracker.js

```diff
@@ -24,7 +24,11 @@ class RageUsage extends ResourceTracker {
   }
 
   onDamage(event) {
-    this.processInvisibleEnergize(SPELLS.RAGE_AUTO_ATTACKS.id, RAGE_PER_MELEE_HIT);
+    if (event.hitType === 2) {
+      this.processInvisibleEnergize(SPELLS.RAGE_AUTO_ATTACKS.id, RAGE_PER_MELEE_HIT * 1.3);
+    } else if (event.hitType === 1) {
```

Comment:
same thing here

```suggestion
    } else if (event.hitType === HIT_TYPE.NORMAL) {
```

or simply

```suggestion
    } else {
```

if there are no other conditions where the hit might (or might not) generate rage.

Reply:
Based on looking at a few logs I think melee hits only give rage if they are of hit types of NORMAL or CRIT. PARRY does not and I have not seen other types of hits in pve logs. So I don't think the if statement can be simplified.